pkg/kubelet/kubelet.go

```diff
@@ -793,7 +794,11 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 
 	klet.appArmorValidator = apparmor.NewValidator(kubeCfg.ContainerRuntime)
 	klet.softAdmitHandlers.AddPodAdmitHandler(lifecycle.NewAppArmorAdmitHandler(klet.appArmorValidator))
-
+	if utilfeature.DefaultFeatureGate.Enabled(features.Accelerators) {
+		klet.gpuManager = nvidia.NewNvidiaGPUManager(klet, klet.dockerClient)
```

Comment:
yeah. Been meaning to add that check. I think it is invalid to enable GPU support when runtime is not docker. With CRI though, would we know the runtime before hand?